source/speech/speech.py

```diff
@@ -1355,6 +1356,18 @@ def _suppressSpeakTypedCharacters(number: int):
 FIRST_NONCONTROL_CHAR = " "
 
 
+def isFocusEditable() -> bool:
+	"""Check if the currently focused object is editable.
+	@return: C{True} if the focused object is editable, C{False} otherwise.
+	@rtype: bool
+	"""
```

Comment:
Please rewrite this to be in sphinx style
```suggestion
	"""Check if the currently focused object is editable.
	:return: ``True`` if the focused object is editable, ``False`` otherwise.
	"""
```